nrf_modem/doc/at_interface.rst

```diff
@@ -1,16 +1,13 @@
 .. _nrf_modem_at:
 
-AT interface
+AT commands
```

Comment:
I reworded this on purpose because and here's why.

This library offers access to all features in the modem, and when the user want to know how to use these features they come to this page.

When they do, what they want to know how is:
- How do I send AT commands?
- How do I use GNSS?
- How do I do delta firmware updates of the modem?

Rather than:
- What is the AT interface?
- What is the GNSS interface?

and when they come looking here, they should see a list of features not of "interfaces" for said features.

The "interface" is just a mechanism by which the feature is used, so I think we should start by talking about features, and then say, this feature is controlled by this interface in the library.

This is much more intuitive to navigate, because for each modem feature there is a dedicated page that introduces the user to the feature and then it details the API that is offered by the library to control said feature.

Reply:
I noticed this when I put the links to the different features in the table in `flavour_features.rst` (first column) and the features of the modem where "GNSS interface" or "AT interface" instead of "Global navigation satellite system", "AT commands".

I would like to have a distinction between the feature in the modem and the API we offer to use that feature, and I would like to start by talking about the feature, and then the API to use it, that is, the user should navigate looking at features not APIs (since this is the conceptual documentation).